### PR TITLE
Exchanges: Convert bespoke Number types to types.Number

### PR DESCRIPTION
**Pull request by gbjk**
## Type of change

- [x] Non-Breaking improvement

## How has this been tested

- [x] go test ./... -race
- [x] golangci-lint run

### CODE REVIEW

**Review by shazbert — APPROVED** [review]
tACK. Nice.

**Review by gloriousCode — APPROVED** [review]
I give this a "10" out of 10. tACK

File: exchanges/okcoin/okcoin_types.go [hunk 1]
```diff
@@ -160,13 +160,13 @@ type WebsocketOrderbookResponse struct {
 // WebsocketOrderBook holds orderbook data
 type WebsocketOrderBook struct {
 	Checksum  int64             `json:"checksum"`
-	Asks      [][2]okcoinNumber `json:"asks"` // [ Price, Quantity, depreciated, number of orders at the price ]
-	Bids      [][2]okcoinNumber `json:"bids"` // [ Price, Quantity, depreciated, number of orders at the price ]
+	Asks      [][2]types.Number `json:"asks"` // [ Price, Quantity, depreciated, number of orders at the price ]
```
Comment (thrasher-) [inline exchanges/okcoin/okcoin_types.go:163]: Despite not being related to this PR, when reviewing these changes we're expecting 4 items as per the comments but only pre-allocating 2. [The API also states 4 items for both REST and websocket](https://www.okcoin.com/api/v5/market/books?instId=BTC-USD). Let me know if you'd like to resolve it in this PR or we can add it as a task for a separate PR

Comment (gbjk) [reply exchanges/okcoin/okcoin_types.go:163]: I'll pass on resolving it in this PR, please Bob 🙏 

@Beadko - Perhaps you'd like to take this one?

**Review by thrasher- — APPROVED** [review]
Nice clean up

File: exchanges/okcoin/okcoin_types.go (hunk 1 above)
Comment (Beadko) [reply exchanges/okcoin/okcoin_types.go:163]: sure

**Pull request merged**